docs/api.yml

```diff
@@ -0,0 +1,226 @@
+          description: |
+            Return object containting list of rows and groups of history entries
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  rows:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        group:
+                          type: string
+                        subgroup:
+                          type: number
+                        start:
+                          type: number
+                        end:
+                          type: number
+                        type:
+                          type: string
+                          enum: [range, point]
+                        content:
+                          type: string
+                  groups:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        id:
+                          type: string
+                        content:
+                          type: number
+                        order:
+                          type: number
+  /job/{job_id}/cancel:
+    post:
+      summary: Cancel the job with the specified job_id
+      parameters:
+        - name: job_id
+          in: path
+          schema:
+            type: string
+            format: uuid
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/empty:
+    post:
+      summary: Empty the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/delete:
+    post:
+      summary: Delete the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queues.json:
+    get:
+      summary: List Queues and their state
+      responses:
+        200:
+          description: |
+            Return object containting list of queues and theis states
```

Comment:
I'm not sure description gives more insight here than the summary

Reply:
```suggestion
            Return object containing a list of queues, including `[failed]`,
            `[finished]` and `[running]` registries, the number of elements
            in them and an url for more details on that queue. 
```